bitnami/tomcat/README.md

```diff
@@ -95,6 +95,7 @@ The following tables lists the configurable parameters of the Tomcat chart and t
 |-----------------------------|-------------------------------------------------------------------------------------------|---------------------------------------------|
 | `replicaCount`              | Specify number of Tomcat replicas                                                         | `1`                                         |
 | `containerPort`             | HTTP port to expose at container level                                                    | `8080`                                      |
+| `containerExtraPorts`       | Extra HTTP ports to expose at container level                                                    | `{}`                                      |
```

Comment:
The ports to open can potentially use any protocol. Therefore, I'd remove the "HTTP" reference:
```suggestion
| `containerExtraPorts`       | Extra ports to expose at container level                                               | `{}`                                      |
```

Reply:
right I missed this ;) (and in example I used AJP)